### PR TITLE
Implement key-In transaction

### PR DESCRIPTION
**Pull request by priyankshah-personal**
Jira Ticket: https://engineering.paypalcorp.com/jira/browse/DTPPHMM-980

UIChanges: 
1. Add PaymentType button in PaymentViewController 
2. Popup for entering card details
3. ActionSheet for selecting Key-In

FileChanges:
1. Add AppStrings File for strings 

Screenshots:
Simulator Screen Shot - iPhone 8 - 2021-03-22 at 08 05 30
Simulator Screen Shot - iPhone 8 - 2021-03-22 at 08 05 38
Simulator Screen Shot - iPhone 8 - 2021-03-22 at 08 05 45

### CODE REVIEW

File: SampleApp/PPHSDKSampleApp/PaymentViewController.swift [hunk 1]
```diff
@@ -29,10 +39,15 @@ class PaymentViewController: UIViewController, PPHRetailSDKAppDelegate {
     var options = PPRetailTransactionBeginOptions.defaultOptions()
     var formFactorArray: [PPRetailFormFactor] = []
     var currencySymbol: String!
+    var paymentType: PaymentTypes?
+    let manuallyEnteredCard = PPRetailManuallyEnteredCard()
+    var manuallyEnteredCardPresent: Bool = false
```
Comment (pranav0705) [inline SampleApp/PPHSDKSampleApp/PaymentViewController.swift:44]: We dont need to implicitly declare variable here

File: SampleApp/PPHSDKSampleApp/PaymentViewController.swift [hunk 2]
```diff
@@ -29,10 +39,15 @@ class PaymentViewController: UIViewController, PPHRetailSDKAppDelegate {
...
     
     // Get the online or offline state from the SDK by calling the "PayPalRetailSDK.transactionManager().getOfflinePaymentEnabled()"
     var offlineMode: Bool = PayPalRetailSDK.transactionManager().getOfflinePaymentEnabled()
     
+    let paymentTypes: [String] = [AppStrings.PaymentOptions.paymentTypeCardReader, AppStrings.PaymentOptions.paymentTypeDigitalCard, AppStrings.PaymentOptions.paymentTypeCash, AppStrings.PaymentOptions.paymentTypeKeyIn, AppStrings.PaymentOptions.paymentTypeCheck, AppStrings.PaymentOptions.paymentTypeQRC]
```
Comment (pranav0705) [inline SampleApp/PPHSDKSampleApp/PaymentViewController.swift:49]: Suggestion: Can we write each index in a newline?

File: SampleApp/PPHSDKSampleApp/PaymentViewController.swift [hunk 3]
```diff
@@ -57,6 +72,103 @@ class PaymentViewController: UIViewController, PPHRetailSDKAppDelegate {
...
+    controller.addTextField { (cardPostalCodeTextField) in
+        cardPostalCodeTextField.keyboardType = .numberPad
+        cardPostalCodeTextField.placeholder = "Enter Postal Code"
+    }
+
+    controller.addAction(UIAlertAction(title: AppStrings.AlertActionButtonTitles.alertActionTitleDone, style: .default, handler: { (action) in
+        let cardNumberTextField = controller.textFields![0]
+        let cardNumberExpiryTextField = controller.textFields![1]
+        let cardCVVTextField = controller.textFields![2]
+        let cardPostalCodeTextField = controller.textFields![3]
```
Comment (pranav0705) [inline SampleApp/PPHSDKSampleApp/PaymentViewController.swift:99]: Can we avoid force wrapping here?

File: SampleApp/PPHSDKSampleApp/PaymentViewController.swift [hunk 4]
```diff
@@ -57,6 +72,103 @@ class PaymentViewController: UIViewController, PPHRetailSDKAppDelegate {
...
+
+                self.btnPaymentType.setTitle(paymentType, for: .normal)
+                }
+            }
+            
+            if index == 3 {
+                alertAction.isEnabled = true
+            } else {
+                alertAction.isEnabled = false
+            }
```
Comment (pranav0705) [inline SampleApp/PPHSDKSampleApp/PaymentViewController.swift:161]: Suggestion: We can write this as alertAction.isEnabled = index == 3 ? true : false

File: SampleApp/PPHSDKSampleApp/PaymentViewController.swift [hunk 5]
```diff
@@ -57,6 +72,103 @@ class PaymentViewController: UIViewController, PPHRetailSDKAppDelegate {
...
+                    self.options?.manualCard = self.manuallyEnteredCard
+                } else if index == 4 {
+                    // self.getManualCheckAmount()
+                    self.paymentType = PaymentTypes.paymentTypeCheck
+                } else if index == 5 {
+                    // self.paymentTypeQRC()
+                    self.paymentType = PaymentTypes.paymentTypeQRC
+
+                self.btnPaymentType.setTitle(paymentType, for: .normal)
+                }
```
Comment (pranav0705) [inline SampleApp/PPHSDKSampleApp/PaymentViewController.swift:154]: Can we use switch here?

File: SampleApp/PPHSDKSampleApp/PaymentViewController.swift (hunk 2 above)
Comment (priyankshah-personal) [reply SampleApp/PPHSDKSampleApp/PaymentViewController.swift:49]: done

File: SampleApp/PPHSDKSampleApp/PaymentViewController.swift (hunk 1 above)
Comment (priyankshah-personal) [reply SampleApp/PPHSDKSampleApp/PaymentViewController.swift:44]: Done

File: SampleApp/PPHSDKSampleApp/PaymentViewController.swift (hunk 3 above)
Comment (priyankshah-personal) [reply SampleApp/PPHSDKSampleApp/PaymentViewController.swift:99]: Done

File: SampleApp/PPHSDKSampleApp/PaymentViewController.swift (hunk 4 above)
Comment (priyankshah-personal) [reply SampleApp/PPHSDKSampleApp/PaymentViewController.swift:161]: Done

File: SampleApp/PPHSDKSampleApp/PaymentViewController.swift (hunk 5 above)
Comment (priyankshah-personal) [reply SampleApp/PPHSDKSampleApp/PaymentViewController.swift:154]: Done

**Review by shupaypal — APPROVED** [review]

**Review by pranav0705 — APPROVED** [review]

**Pull request merged**